articles/native-platforms/ios-objc/05-linking-accounts.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Linking Accounts
+description: This tutorial will show you how to link different user profiles, allowing different ways of signing in into a single profile.
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
```

Comment:
@pablolvillar @sebacancinos 
Please change this to fork, like the rest of the ios-objc docs.